src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FeeStep.tsx

```diff
@@ -41,13 +44,8 @@ export const FeeStep = ({ isLastStep }: FeeStepProps) => {
   // use the fee data from the actual quote in case it varies from the theoretical calculation
   const affiliateBps = useAppSelector(selectActiveQuoteAffiliateBps)
 
-  const feeModel = 'SWAPPER'
```

Comment:
The intent was to generalize this, but `<FeeStep />` is used exclusively in swapper so we can leverage the userCurrency selector, which selects precisely that, the *actual* fee (after discount, the same that's used at rate details time), already prorated to user currency, and without needing to use the more generalized `selectCalculatedFees()`